doc/modules/sorters.rst

```diff
@@ -285,27 +285,26 @@ Running several sorters in parallel
 
 The :py:mod:`~spikeinterface.sorters` module also includes tools to run several spike sorting jobs
 sequentially or in parallel. This can be done with the
-:py:func:`~spikeinterface.sorters.run_sorters()` function by specifying
+:py:func:`~spikeinterface.sorters.run_sorter_jobs()` function by specifying
 an :code:`engine` that supports parallel processing (such as :code:`joblib` or :code:`slurm`).
 
 .. code-block:: python
 
-    recordings = {'rec1' : recording, 'rec2': another_recording}
-    sorter_list = ['herdingspikes', 'tridesclous']
-    sorter_params = {
-                    'herdingspikes': {'clustering_bandwidth' : 8},
-                    'tridesclous': {'detect_threshold' : 5.},
-                    }
-    sorting_output = run_sorters(sorter_list, recordings, working_folder='tmp_some_sorters',
-                                 mode_if_folder_exists='overwrite', sorter_params=sorter_params)
+    # here we run 2 sorters on 2 different recordings = 4 jobs
+    recording = ...
+    another_recording = ...
+
+    job_list = [
```

Comment:
I was going to suggest using the format that scikt-learn uses for exhaustive grid searh which should save you some typing:

https://scikit-learn.org/stable/modules/grid_search.html#exhaustive-grid-search

But maybe it does not work that well with some arguments like folder.

Nevertheless I suggest to take a look the strategies that they use. It is good not to reinvent the wheel and also to make use of the patterns and formats that our users already know (as scikit-learn is popular within the scientific comunity)

https://scikit-learn.org/stable/modules/grid_search.html#exhaustive-grid-search

Reply:
I think we do not want any pattern here. We want to process a flat list of run_sorter() with deiffernt engine.
I would not add smantic of looping here and combinaison here.
What we implement here is the job dispatch. If someone want a job list creation that have grid combinason semantic we can do it but there is no need for now.

Reply:
Yep, makes sense to me. Just adding this as relevant to the discussion for the future.